minimesos/src/main/java/com/containersol/minimesos/main/CommandPs.java

```diff
@@ -0,0 +1,60 @@
+package com.containersol.minimesos.main;
+
+import com.containersol.minimesos.MinimesosException;
+import com.containersol.minimesos.cluster.ClusterRepository;
+import com.containersol.minimesos.cluster.MesosCluster;
+import com.containersol.minimesos.mesos.MesosClusterContainersFactory;
+import com.containersol.minimesos.state.Framework;
+import com.containersol.minimesos.state.State;
+import com.containersol.minimesos.state.Task;
+
+import java.io.PrintStream;
+
+/**
+ * Lists tasks on the cluster
+ */
+public class CommandPs implements Command {
+
+    private ClusterRepository repository = new ClusterRepository();
+
+    private PrintStream output = System.out;
+
+    public CommandPs(PrintStream output) {
+        this.output = output;
+    }
+
+    public CommandPs() {
+        this.output = System.out;
+    }
+
+    @Override
+    public boolean validateParameters() {
+        return true;
+    }
+
+    @Override
+    public String getName() {
+        return "ps";
+    }
+
+    @Override
+    public void execute() throws MinimesosException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'com.containersol.minimesos.MinimesosException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)